verticales/simplifions/grist_v2_manager.py

```diff
@@ -88,3 +115,78 @@ def get_and_format_grist_v2_data(self, ti):
         logging.info("grist_tables_for_filters done.")
 
         ti.xcom_push(key="grist_tables_for_filters", value=grist_tables_for_filters)
+
+    def watch_grist_data(self, ti):
+        """
+        Watch Grist data for changes and log the current state.
+        This method monitors the Grist tables and provides information about data changes.
+        """
+        logging.info("Grist data watch started")
+        all_tables = self._request_all_tables()
+        time_delta = 24  # in hours
+        cutoff_timestamp = (datetime.now() - timedelta(hours=time_delta)).timestamp()
+        tables_with_modified_rows = {}
+
+        # Get all tables data
+
+        for table in all_tables:
+            logging.info("\n")
+            logging.info(f"Table: {table['id']}")
+            table_columns = self._request_table_columns(table["id"])
+
+            if "Modifie_le" in [column["id"] for column in table_columns]:
+                all_rows = self._request_grist_table(table["id"])
+                logging.info(f"> Found {len(all_rows)} rows total")
+
+                modified_rows = [
+                    row
+                    for row in all_rows
+                    if row["fields"].get("Modifie_le")
+                    and row["fields"]["Modifie_le"] > cutoff_timestamp
+                ]
+
+                if modified_rows and len(modified_rows) > 0:
```

Comment:
`if my_list` is the same as `if len(my_list) > 0` so it's redundant
```suggestion
                if modified_rows:
```